Saturation/Splitter.cpp

```diff
@@ -396,39 +396,54 @@ void SplittingBranchSelector::updateSelection(unsigned satVar, SATSolver::VarAss
   SplitLevel posLvl = _parent.getNameFromLiteral(SATLiteral(satVar, true));
   SplitLevel negLvl = _parent.getNameFromLiteral(SATLiteral(satVar, false));
 
+  bool posUsed = _parent.isUsedName(posLvl);
+  bool negUsed = _parent.isUsedName(negLvl);
```

Comment:
Does it make sense to add an assertion if these are mutually exclusive or is this a SAT solver invariant?

Reply:
I think one of them implies the other, but they are not mutex. Assertion can be added, but I don't remember by heart which one :(

Reply:
Actually, this also depends on SplittingAddComplementary. I don't think the assertion would be very legible even if I managed to write it down.